api_keys/admin.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+
+from .models import ApiKey
+
+
+@admin.register(ApiKey)
+class ApiKeyAdmin(admin.ModelAdmin):
+    list_display = ("preview", "workspace", "created_by", "created_at")
```

Comment:
`search_fields` - workspace__name, workspace__id, created_by__display_name, created_by__uid, preview

Reply:
+ do some research - how we do it in other places